src/components/ViewList.js

```diff
@@ -45,8 +59,9 @@ let ViewList = ({
             <div
               key={"list-" + company.name}
               className={"col-name-item" + (index % 2 === 0 ? " even" : " odd")}
+              onClick={() => targetEdit(company.name)}
             >
-              {company.name}
+              <Link to="/edit">{company.name}</Link>
```

Comment:
Add Link to edit page; user can click a company name to be routed to the edit page with that company pre-populated in the form.